rootfs/etc/nginx/lua/plugins.lua

```diff
@@ -22,6 +21,11 @@ local function load_plugin(name)
 end
 
 function _M.init(names)
+  if nkeys(plugins) > MAX_NUMBER_OF_PLUGINS then
```

Comment:
`nkeys(plugins)` will always be `0` here since `load_plugin` is not called yet.

I think the validation should happen in the below loop. And we don't need `nkeys`, we can simply increment a counter after each load and then check if we loaded `MAX_NUMBER_OF_PLUGINS` plugins.

Reply:
Sorry for my carelessness.